app/models/translation.rb

```diff
@@ -10,6 +10,23 @@ class Translation < ActiveRecord::Base
 
   @@root_folder_name = 'public/srt/'
 
+  # Maps from integer representation in the database to status symbol
+  @@STATUS_HASH = {0 => :incomplete, 1 => :complete, 2 => :complete_with_priority}
```

Comment:
Minor stylistic nitpick: for cleanliness, use spaces as padding inside large hashes.
```ruby
{ 0 => :incomplete, 1 => :complete, 2 => :complete_with_priority }
```